2021/OffSeason2021/src/main/cpp/subsystems/Climber.cpp

```diff
@@ -125,29 +125,31 @@ void Climber::RaiseClimberWithJoysticks(frc::XboxController *joystick)
 
     yCLValue = joystick->GetY(frc::GenericHID::JoystickHand::kLeftHand);
 
+    if (yCLValue > -0.1 && yCLValue < 0.1)
+    {
+        spdlog::info("Climber Stopped");
+        SetBrakeSolenoid(CL_BRAKE_LOCKED);
+    }
+    else
+    {
+        SetBrakeSolenoid(CL_BRAKE_UNLOCKED);
+        // If joystick is above a value, climber will move up
+        if (yCLValue > 0.1)
+            motorOutput = m_upSpeed;
+
+        // If joystick is below a value, climber will move down
+        else if (yCLValue < -0.1)
+            motorOutput = m_downSpeed;
+    }

```

Comment:
Don't forget to square the motor output and keep the original sign after squaring

Reply:
Will do that as soon as I can!

Reply:
Don't wait for the squaring to get this PR approved and in. Treat that as a separate issue. This is exactly what I mean about getting a base level of functionality, making a commit, creating a PR, and getting it merged. We don't want to be waiting for the final version when we have one that gives us valuable base functionality.